test/builder/utila2.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package builder
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	vmopv1 "github.com/vmware-tanzu/vm-operator/api/v1alpha2"
```

Comment:
> ... one way to future proof this is ...

I don't see the future proofing here. The transition to v1a2 is made more difficult & disruptive since it addresses years of hacks and lessons learned for v1a1 but for future v1a3+ and later versions, I expect the schema changes to be incremental & relatively small. Maybe I am misunderstand what you're proposing here, but using `vmopv2` would make a later `vmopv3` diff much larger since numerous lines need to change instead of the just import in most cases. The use of a common import alias was an intentional choice at the start of the v1a2 work.